src/plugins/ui_actions/public/components/error_configure_action/action_identifier.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { EuiCode } from '@elastic/eui';
+import { ActionInternal } from '../../actions';
+
+export interface ActionIdentifierProps {
+  action: ActionInternal;
+}
+
+export const ActionIdentifier: React.FC<ActionIdentifierProps> = ({ action }) => (
```

Comment:
Oh, right, it is not.

We wanted to wrap <actionFactory.ReactCollectConfig /> into error boundery (to notify devs about errors in their collect conig component)  but it turned it would require changes in uiComponent util and we postponed to 7.8.

I think this should be removed for now